CHANGELOG.md

```diff
@@ -51,6 +51,15 @@ Our versioning strategy is as follows:
 
 ### 🧹 Chores
 
+## 22.1.1
+
+### 🎉 New Features & Improvements
+
+NOTE: API was changed, preview data provides a new parameter `sc_layoutKind` therefore, please make the necessary updates to the nextjs template files as shown in the following PR to experience a smooth upgrade.
+
+* `[sitecore-jss]``[sitecore-jss-nextjs]` Pass `sc_layoutKind` to GraphQLEditingService request header to support shared/final editing layouts ([#1907](https://github.com/Sitecore/jss/pull/1907))
+  * NOTE: `GraphQLRequestClient`'s  `request` method now supports dynamic headers.
```

Comment:
This should be extracted as a separate note but with the reference to the same PR. It's a separate new feature
* `[sitecore-jss]` `GraphQLRequestClient` `request` method now supports dynamic headers based on specific request ([#1907](https://github.com/Sitecore/jss/pull/1907))
  ```ts
  client.request(query, variables, { headers })
  ```